experiments/ssl4eo/flops.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import timm
+from deepspeed.accelerator import get_accelerator
+from deepspeed.profiling.flops_profiler import get_model_profile
+
+models = ["resnet18", "resnet50", "vit_small_patch16_224"]
+num_classes = 14
+in_channels = 11
+batch_size = 64
+patch_size = 224
+input_shape = (batch_size, in_channels, patch_size, patch_size)
+
+for model in models:
+    print(f"Model: {model}")
+
+    m = timm.create_model(model, num_classes=num_classes, in_chans=in_channels)
+
+    # Calculate memory requirements of model
+    mem_params = sum([p.nelement() * p.element_size() for p in m.parameters()])
+    mem_bufs = sum([b.nelement() * b.element_size() for b in m.buffers()])
+    mem = (mem_params + mem_bufs) / 2**20
```

Comment:
Where does 2**20 come from? I thought MB was 10**6?

Reply:
$2^{20} \approx 10^6$. I've seen both definitions of MB, not sure which is more "official".

Reply:
I'll keep it as is for now so we don't have to change the paper, but maybe we should change it to $10^6$ if we use the same script for future papers.

Reply:
**I would if it was really easy ;) -- merged